guides/common/attributes-satellite.adoc

```diff
@@ -8,6 +8,7 @@
 :BaseURL: {RHDocsBaseURL}red_hat_satellite/{ProjectVersion}/html-single/
 
 // URLs (published on Red Hat Portal)
+// IMPORTANT: When you change any guide URLs, also update the json file in downstream
```

Comment:
Given today we have multiple places, please also list https://github.com/theforeman/foreman-documentation/blob/master/guides/upstream_filename_to_satellite_link.json and https://github.com/RedHatSatellite/foreman_theme_satellite/blob/develop/lib/foreman_theme_satellite/documentation.rb